docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,283 @@
+contributors, as well as to submit a request of co-authorship for special contributions. The same rules and methods
+as for the standard releases are applied here.
+
+
+For the first LTS release, the `V1.0`, all contributors from the beginning of the project will be co-authors per default.
+As the DCO has not be applied for the past contributions, the best will be done to contact all contributors in order to
+exercise their right to withdraw ("OptOut" scheme), to update their personal information, and to add authors with special
+contributions. The order of the authors is 'The Gammapy team', followed by the list of contributors in alphabetical order.
+
+General Gammapy publications
+----------------------------
+
+This product aims to describe the project and/or the library as a whole and will be more of the case associated with the
+publication of an LTS release. As it targets a wide community, the following scheme is used.
+
+1. List of authors
+    a/ By default, 'The Gammapy team' is mentioned first, then the Lead Developers, then the past Lead Developers since
+    the last general Gammapy publication, then the list of contributors of each release since the last Gammapy publication.
+
+    b/ If the editorial rules of the targeted journal permit it, the scheme used by the Astropy project
+    (e.g. `Astropy V2.0 <https://arxiv.org/pdf/1801.02634.pdf>`_) should be used in priority:
+
+    - 'The Gammapy team', and the list of primary paper contributors in order of contribution agreed per consensus with the development team, as '(Primary Paper Contributors)',
+    - the members of the Coordination Committee, as '(Gammapy Coordination Committee)',
+    - the list of contributors of the associated LTS ordered by alphabetical order, as '(Gammapy Contributors)'
+
+    In this case, a comment on the author list composition should be added. Extracted from the Astropy project,
+    one can precise:
+
+    "The author list has three parts: the authors that made significant contributions to the writing of the paper,
+    the members of the Gammapy Project Coordination Committee, and contributors to the Gammapy Project in alphabetical
+    order. The position in the author list does not correspond to contributions to the Gammapy Project as a whole. A more
+    complete list of contributors to the core package can be found in the package repository and at the Gammapy team
+    webpage."
+
+2. Corresponding author
+    It is the 'Gammapy Coordination Committee', associated with its usual mailing list (GAMMAPY-COORDINATION-L@IN2P3.FR).
+
+3. Acknowledgement
+    One has the freedom to precisely mention acknowledgements associated with the publication. In practice, it is
+    recommended to precise the grants or fellowships given to some authors. One should in any case always acknowledge
+    the Astropy project, to which we are affiliated, and our mandatory external libraries (e.g. numpy, scipy, matplotlib,
+    iminuit).
```

Comment:
This is probably a separate issue, but should there be a general Gammapy Acknowledgement? E.g. on the home page?

E.g. my funding agency does in principle require this, and I guess this holds true for many other people

Reply:
I agree we could mention here the possibility of maintaining an independent list of acknowledgments e.g. on the webpage.